PowerShell/JumpCloud Module/JumpCloud.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
     <id>JumpCloud</id>
-    <version>1.18.4</version>
+    <version>2.0.0.5839-202208041552</version>
```

Comment:
Nuspec versions will be updated as we build the module, when we release this should be changed to the Module.PSD1 version (2.0.0)